### PR TITLE
update and fix problem in API doc.

### PR DESCRIPTION
**Pull request by perhapszzy**
The parameters in bounding box should be [y_min, x_min, y_max, x_max]

### CODE REVIEW

File: tensorflow/g3doc/api_docs/python/functions_and_classes/shard8/tf.image.draw_bounding_boxes.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ bounding box coordinates are floats in `[0.0, 1.0]` relative to the width and
 height of the underlying image.
 
 For example, if an image is 100 x 200 pixels and the bounding box is
-`[0.1, 0.5, 0.2, 0.9]`, the bottom-left and upper-right coordinates of the
```
Comment (vrv) [inline tensorflow/g3doc/api_docs/python/functions_and_classes/shard8/tf.image.draw_bounding_boxes.md]: You should update the original code in image_ops.py, the .md files are autogenerated.

**Review by vrv — CHANGES_REQUESTED** [review]

File: tensorflow/g3doc/api_docs/python/functions_and_classes/shard8/tf.image.draw_bounding_boxes.md (hunk 1 above)
Comment (perhapszzy) [inline tensorflow/g3doc/api_docs/python/functions_and_classes/shard8/tf.image.draw_bounding_boxes.md]: changed this at image_ops.cc.

**Review by vrv — APPROVED** [review]

**Comment by vrv** [comment]
@tensorflow-jenkins test this please

**Comment by martinwicke** [comment]
Last test failure was infrastructure. Overriding.

**Pull request merged**